sdk/ml/azure-ai-ml/azure/ai/ml/operations/_feature_store_operations.py

```diff
@@ -231,6 +262,26 @@ def begin_update(
                 if not materialization_identity:
                     raise ValidationError("Materialization identity is required to setup offline store connection")
 
+        if online_store and online_store.type != ONLINE_MATERIALIZATION_STORE_TYPE:
+            raise ValidationError("online store type should be redis")
+
+        if online_store and rest_workspace_obj.feature_store_settings.online_store_connection_name:
+            existing_online_Store_connection = self._workspace_connection_operation.get(
+                resource_group,
+                feature_store.name,
+                rest_workspace_obj.feature_store_settings.online_store_connection_name,
+            )
+
+            if existing_online_Store_connection:
```

Comment:
Lets make this camel case existing_online_Store_connection --> existing_online_store_connection

Reply:
fixed